src/main/java/co/rsk/federate/signing/hsm/advanceblockchain/HSMBookkeepingService.java

```diff
@@ -113,9 +115,15 @@ public boolean isStarted() {
         return started;
     }
 
-    private Block getHsmBestBlock() throws HSMClientException {
+    private Block getHsmBestBlock() throws HSMClientException, BlockStoreException {
```

Comment:
What happens with the BlockStoreException? Is it caught at any point or does it bring down the node?

Reply:
Yes It Caught [Here](https://github.com/rsksmart/powpeg-node/blob/b0c89b2840d8fb33e822f7a0b9ef63771d474628/src/main/java/co/rsk/federate/signing/hsm/advanceblockchain/HSMBookkeepingService.java#L175), And Doesnt Bring Down The Node

Reply:
I see, thanks